stdlib/2and3/difflib.pyi

```diff
@@ -2,13 +2,20 @@
 
 # TODO: Support unicode in Python 2?
 
+import sys
 from typing import (
     TypeVar, Callable, Iterable, Iterator, List, NamedTuple, Sequence, Tuple,
     Generic, Optional
 )
 
 _T = TypeVar('_T')
 
+Match = NamedTuple('Match', [
```

Comment:
I just tried, in 3.5 mode mypy gives
```
stdlib/2and3/difflib.pyi:19: error: NamedTuple class syntax is only supported in Python 3.6
```
Somebody (David I think?) explained that while syntactic 3.6 features work in all stubs, language-level 3.6 features don't. I'm not sure that distinction really makes sense to me, but I guess what it means concretely is that typed_ast processes stubs in 3.6 mode, but mypy's semantic analyzer uses 2.7 mode if mypy is in 2.7 mode, and the semantic analyzer is the place that rejects NamedTuple class syntax.